daml/splice-amulet/daml/Splice/AmuletRules.daml

```diff
@@ -1053,22 +1053,21 @@ summarizeTransfer sender openRound transferConfigAmulet inp preprocessedOutputs
       amuletPrice = openRound.amuletPrice
 
 issueRewards : RewardsIssuanceConfig -> TransferContextSummary -> Party -> Party -> Optional [AppRewardBeneficiary] -> TransferSummary -> Update ()
-issueRewards config csum sender provider beneficiaries summary
-  | usageFees <= 0.0 = pure ()
-  | otherwise = do
-      when (config.issueValidatorRewards) $
-        void $ create ValidatorRewardCoupon with
-          dso = csum.openRound.dso
-          user = sender
-          amount = usageFees
-          round = csum.openRound.round
-
-      if config.issueAppRewards
-        then do
-          let beneficiaries' = fromOptional [AppRewardBeneficiary provider 1.0] beneficiaries
-          validateAppRewardBeneficiaries beneficiaries'
-          let groupedBeneficiaries = Map.fromListWithR (+) (map (\b -> (b.beneficiary, b.weight)) beneficiaries')
-              totalAmount = usageFees + extraAppReward
+issueRewards config csum sender provider beneficiaries summary = do
+    when (config.issueValidatorRewards && usageFees > 0.0) $
```

Comment:
TODO: write tests for this case, and double-check all `issueRewards` usages wrt doing the right thing.